contents/blog/introducing-hogql.md

```diff
@@ -0,0 +1,98 @@
+---
+date: 2023-03-02
+title: "Introducing HogQL, SQL and Custom insights for PostHog"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+category: PostHog news
+author:
+  - joe-martin
+tags:
+  - Product updates
+  - Release notes
+featuredImage: ../images/blog/hog_ql.png
+featuredImageType: full
+---
+
+Today, we're excited to announce a major new feature which we're making available to all users as a public beta on PostHog Cloud. It's called HogQL and it's a new query language which makes it easier to interrogate your data directly.
```

Comment:
The intent of this line reads as if we are "launching a new query language". However what we're really launching is the ability for anyone to make SQL queries against data stored in PostHog. I think that fact should be the headline feature. 

To make those SQL queries, you can:
- Write actual SQL and save it as an insight
- Use the "event explorer" view with columns that you can customise with "SQL expressions" (aka "chunks of SQL")
- Use "SQL expressions" in existing insights.

Now, our dialect of SQL is called HogQL, and the terms "SQL" and "HogQL" are interchangeable in a technical context. However emphasising that it's a new language makes it feel like a hurdle that I need to overcome.

Thus, saying we're "launching SQL support" will directly communicate the essence of what this change brings. Saying that we're "launching HogQL" makes you just wonder "What is that now?".

I wonder if we should just rename the blog post to "Announcing SQL support"?

Reply:
Running in some changes to facilitate this.